package.json

```diff
@@ -26,6 +28,7 @@
     "gatsby-plugin-sass": "^4.8.0",
     "gatsby-plugin-sharp": "^3.14.1",
     "gatsby-plugin-sitemap": "^4.4.0",
+    "gatsby-plugin-typegen": "^2.2.4",
```

Comment:
Should these be moved to `devDependencies`?

Reply:
done :)